### DISCUSSION TITLE
Change configs from a mod in a curseforge modpack

### DISCUSSION THREAD

**Discussion by Goten87** [itzg/docker-minecraft-server#2338] [Q&A]
Hello i run a ATM8 pack with docker on a ubuntu server
and i want to change some configs to what i want 
(AE2 channels off)
is there a way i can do that in the docker-compose file so then when i update the pack it will not overwrite it again so that i need to change it again?

**Comment by itzg** [comment]
I'll research if the modpack overrides mechanism will leave existing/modified files untouched.

**Reply by Goten87** [reply]
i just updated the ATM8 pack on my test server to latest and i think it kept the config alone settings are still the same (now i dont know what it will do if there will be a change to the settings from the modpack part)

**Reply by itzg** [reply]
> now i dont know what it will do if there will be a change to the settings from the modpack part

It sounds like it worked as expected, but what do you mean by your remaining doubts?

**Reply by Goten87** [reply]
what i mean is what if the mod pack people change something in the configs for lets say AE2 will it then overwrite the config file

**Reply by itzg** [reply]
Thanks, now I understand. I still owe some research on that.
